src/java.xml/share/classes/com/sun/org/apache/xml/internal/serialize/HTMLdtd.java

```diff
@@ -438,8 +438,8 @@ private static void initialize()
     private static void defineEntity( String name, char value )
     {
         if ( _byName.get( name ) == null ) {
-            _byName.put( name, new Integer( value ) );
```

Comment:
Please make sure @JoeWang-Java reviews the XML changes; there are complications in some areas where the master code is upstream of the code in the JDK.

Reply:
JAXP was subsumed by JSR 379 in Java SE 9.